File: Algorithms/LinearSearch.py
### Fun fact with PY
## Define a function 

def LinearSearch(L,key):  
  for i in range(len(L)):
    if L[i] == key:
      return i
  retrun "Not Found"

if __name__ == "__main__":  ##
  list1 = [0,2,1,4,5]
  k = 5
  print(LinearSearch(list,k))  # found in index 4

File: Js/Algorithms/LinearSearch.js
## Fun Fact with Javascript 

function L(l,key){ 
 
  for(i=0;i<l.length;i++){
   
    if(l[i]==key){
      return i;
     
    }
  }
}

let list = [1,3,5,8,0];
key = 0;
console.log("Search key found in index",L(list,key));   #found in index 4
